protocols/isis/server/server.go

```diff
@@ -81,3 +83,13 @@ func netsCompatible(nets []*types.NET) bool {
 
 	return true
 }
+
+// GetInterfaces gets all configured interfaces
+func (s *Server) GetInterfaces() []string {
```

Comment:
I'm wondering if this should be called `GetInterfaceNames` as that's what we return back. As all other public methods only deal with the names it's consistent though, so no strong feelings.

Reply:
Fixed.